source/examples/react/six-app/src/utils/prop-or.ts

```diff
@@ -1,5 +1,11 @@
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const byKey = (k: string) => <T extends unknown>(o = {}, defaults: T): T => (o as any)[k] || defaults;
+export const byKey =
```

Comment:
Possibly an issue with the code formatter, we need to align it so that those changes are minimised.

Reply:
Reverted the formatting changes. 

Agree that we need to align the code formatting. Currently, there are multiple prettier config files and it seems that not all the files are formatted by prettier either. I suggest to have only one prettier config in the root and format all the files in one commit. But lets do this in another task.

Reply:
I think it would be great to apply a global prettier config and then once done we format everything. But as you suggested let's do this in a separate PR.